docs/faq.md

```diff
@@ -182,14 +187,14 @@ h('div', {
 ```
 
 We only have one type of hook as maintaining both life cycles
-  seperately is very complex when it can simply be done at
+  separately is very complex when it can simply be done at
```

Comment:
I didn't want to 'touch' that section further because then pretty much the whole of the document will have to be changed because that 'indent all but the first line' is used. I haven't really seen this style before and I'm not sure if there's a valid reason for it? 

Usually with text if anything the first line is indented and the others are not. This require less effort too when adjusting text as well.